### DISCUSSION TITLE
Order PCBs somewhere

### DISCUSSION THREAD

**Discussion by 4920441** [stephan192/hoermann_door#23] [Ideas]
Hi,
I'd like to let some of the PCBs being manufactured  from PCBWays or simliar.
Does anyone has interest in  taking part for the order?   I don't wanna make any profit from this,  I think the project is really cool and the original  software does suck a lot.....

I ordered a couple of unassembled PCBs the last years, that worked really well, but I never  ordered a complete assembled pcb there - if anyone has more information about this process, it'd be really appreciated.

Any thoughts?

Cheers

**Comment by kanzlerac** [comment]
Hi, 
I have the same intention and same question (or lack of knowledge) regarding ordering. Does the gerber file have all information for pcb and assembly? 
If I did it correctly, PCBWAY would cost 35 $ for 5 pieces, JLCPCB 34,86 $. Can anyone confrim?

Where are you from? I'm from Germany. 

Christian

**Comment by Gamogamer** [comment]
I have my files at home, I had them made at jlcpcb with partial assembly. I also still have the boards because smart me didn't check properly which supramatic they have. I did however redesign the power delivery because the parts weren't available

**Comment by 4920441** [comment]
Hi,  

I am also currently in Germany. So, "internal shipping" wouldn't be a financial problem :-)

As far as I know are the Gerber files only for routing, vias and drilling instructions.

There has to be a separate file for the parts and the placement of the parts, if I remember correctly.... its now about 25 years ago  I 
did anything like this.... sooooo  don't take my word too serious....

>>  If I did it correctly, PCBWAY would cost 35 $ for 5 pieces, JLCPCB 34,86 $. Can anyone confrim?

Already with the parts placed?  (the 'raw' PCBs should be way less, though - or, the price does not change if you take 5 or 15 pcbs, then sound like the correct amount as well)

Cheers

4920441

**Comment by ralf-e** [comment]
At the end of last year I tried to order assembled boards with the git data at jlcpcb. After the upload, the placement didn't fit (neither the exact location nor the orientation of the components). This can still be adjusted manually there. However, after some of the components were not available there and the price was a lot higher than getting and assembling myself, I stopped and just ordered the circuit boards. I still have 2 of these left... 1€/piece + shipping from Germany

Edit: Both PCBs have been dispatched

**Reply by 4920441** [reply]
I'd like to take one.  How do we get together?

This is a disposable email address of me...:

Voo4Tahthiech3kiM9eif7Xa7eeghiPh@softcell.org

Mail me and I reply with my real one :-)

Cheers

4920441

**Reply by kovlo** [reply]
@ralf-e , if you have one left, I'd be happy to buy it!
I assume I can use it to my Lineamatic gate drive.
Please email me as well: oaz34t1tz@mozmail.com

**Comment by MrBlue90** [comment]
I'm also interested to receive a pre-assembled PCB (and cable) which I can connect directly to my Supramatic E3.
I live in the Netherlands, but really close to the German border if that would help.

Does someone still have such a board available?

**Reply by hvorragend** [reply]
I am also looking into this topic for the first time. Is it possible to buy the boards ready assembled somewhere?

**Comment by Ivy-Mo** [comment]
I can help you order PCBA & PCB. As long as you send the Gerber file & BOM list to this email : ivy@wellerpcb.com , I can give you a quote. But please note that if you only order simple PCB , the minimum order is 10pcs.

**Comment by 14yannick** [comment]
I have some boards left you can have a look [here](https://www.tindie.com/products/14yannick/uapbridge_pic16/)

**Reply by surfmk** [reply]
Hi Yannick, I would be very interested to find a PCB for Stephan's Hoermann Door project (Mine is E3). If you know anyone who can help, I'd love to use this way of controlling my garage door. Thanks in advance.

**Reply by 14yannick** [reply]
Hi, I have no more board and I will also not assemble more of them. I would recommend you to follow the ESP only way. Have a look [there](https://github.com/14yannick/hoermann_door). For this you can go with simple of the shelf component like an M5 Stack atom S3 with a rs 485 base or tail. with this you just have to add a rj12 cable and attach it to the M5stack terminal. Pinout is on the github and if you have a multimeter to check voltage your are 100% save.  If you are not confortable with this you can have preassembled boards from [tysonpower](https://shop.tynet.eu/)
